.travis.yml

```diff
@@ -11,10 +11,7 @@ language: php
 sudo: false
 
 php:
-  - 5.5
```

Comment:
These should not be removed? We can't test only on 7, it's unstable and has potentially different behavior.